### DISCUSSION TITLE
using SignaturePad component on a step

### DISCUSSION THREAD

**Discussion by momla** [Megabit/Blazorise#5203] [Q&A]
Hi, when i use the signaturepad component on a step but it doesn't work if i place it after the first step. When i put it on the first step, it work. It seems to be a render problem. Should the Task.completedTask (OnSelectedStepChanged) affect the render of the component?

**Comment by stsrki** [comment]
Hi, it's hard to tell what might be wrong. Can you open new issue and create a code sample so that we can reproduce it?

**Comment by momla** [comment]
Hi ,
thanks for support.
After update , now signaturepad work but when i release mouse, it shows 2 signs.
Please see capture

[cid:66b577e8-9a1c-4829-a244-efe62bc2f792]
________________________________
De : Mladen Macanović ***@***.***>
Envoyé : mercredi 27 décembre 2023 09:13
À : Megabit/Blazorise ***@***.***>
Cc : momla ***@***.***>; Author ***@***.***>
Objet : Re: [Megabit/Blazorise] using SignaturePad component on a step (Discussion #5203)


Hi, it's hard to tell what might be wrong. Can you open new issue and create a code sample so that we can reproduce it?
